scripts/workflows/wf.yml

```diff
@@ -0,0 +1,81 @@
+steps:
+- id: build-img
+  uses: docker://docker:19.03.10
+  dir: /workspace/scripts/perf/docker
+  args: [build, --tag=spdk, .]
+
+- id: prepare
+  uses: docker://spdk
+  skip_pull: true
+  runs: [bash, -ex, -c]
+  args:
+  - |
+    nvme format /dev/nvme0 --ses=2 -n=1
+    echo 512 > /proc/sys/vm/nr_hugepages
+    sysctl -w vm.nr_hugepages=512
+
+- id: run-read-benchmark
+  uses: docker://spdk
+  skip_pull: true
+  runs: [bash, -ex, -c]
+  args:
+  - |
+    export RW=read IOD=256
+    fio scripts/workflows/precondition.fio
+
+    export RW=randread IOD=256 MIX=100
+    LD_PRELOAD=/root/spdk/examples/bdev/fio_plugin/fio_plugin \
+    fio \
+      --output-format=json \
+      --output=scripts/workflows/results/bdev_read.json \
+      scripts/workflows/spdk_jobfile.fio
+    fio \
+      --output-format=json \
+      --output=scripts/workflows/results/libaio_read.json \
+      scripts/workflows/libaio_jobfile.fio
+
+- id: run-write-benchmark
+  uses: docker://spdk
+  skip_pull: true
+  runs: [bash, -ex, -c]
+  args:
+  - |
+    export RW=write IOD=128
+    fio scripts/workflows/precondition.fio
+
+    export RW=randwrite IOD=32 MIX=0
+    LD_PRELOAD=/root/spdk/examples/bdev/fio_plugin/fio_plugin \
+    fio \
+      --output-format=json \
+      --output=scripts/workflows/results/bdev_write.json \
+      scripts/workflows/spdk_jobfile.fio
+    fio \
+      --output-format=json \
+      --output=scripts/workflows/results/libaio_write.json \
+      scripts/workflows/libaio_jobfile.fio
+
+- id: run-mix-benchmark
+  uses: docker://spdk
+  skip_pull: true
+  runs: [bash, -ex, -c]
+  args:
+  - |
+    export RW=readwrite IOD=256 MIX=70
+    LD_PRELOAD=/root/spdk/examples/bdev/fio_plugin/fio_plugin \
+    fio \
+      --output-format=json \
+      --output=scripts/workflows/results/libaio_mix.json \
+      scripts/workflows/spdk_jobfile.fio
+    fio \
+      --output-format=json \
+      --output=scripts/workflows/results/libaio_mix.json \
+      scripts/workflows/libaio_jobfile.fio
+
+- id: plot-results
+  uses: docker://jupyter/scipy-notebook:latest
+  dir: /workspace/scripts/workflows/results
+  runs: [python3]
+  args: [results.py]
+  env:
+    CHOWN_HOME: 'yes'
+    CHOWN_EXTRA_OPTS: -R
```

Comment:
I think these two variables are not having an effect, since the entrypoint is python3 instead of the one defined in the image